docs/user/guides/setup-single-sign-on-google-email.rst

```diff
@@ -0,0 +1,70 @@
+How to setup Single Sign-On (SSO) with Google Workspace
+=======================================================
+
+.. include:: /shared/admonition-rtd-business.rst
+
+This how-to guide will provide instructions on how to enable :abbr:`SSO (Single Sign-on)` with Google Workspace.
+If you want more information on this feature,
+please read :doc:`/commercial/single-sign-on`
+
+Prerequisites
+-------------
+
+Organization permissions
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+To change your Organization's settings,
+you need to be an *owner* of that organization.
+
+You can validate your ownership of the Organization with these steps:
+
+* Navigate to :guilabel:`Username dropdown` > :guilabel:`Organizations` > :guilabel:`<Organization name>`
+* Look at the **Owners** UI elements on the right menu.
+
+If you'd like to to modify this setting and are not an owner,
+you can ask an existing organization owner to follow this guide.
+
+User setup
+~~~~~~~~~~
+
+Using this setup,
+all users who have access to the domain you configure will be granted a subset of permissions on your organization automatically on account creation.
+Other users can also be added to your organization and given access in an ad hoc manner.
```

Comment:
```suggestion
.. tip::

   You can still add outside collaborators and manage their access. Organizations may for instance wish to give read-access to onboarding documents or grant temporary write access to external consultants.
   
.. seealso::

   :doc:`/commercial/sharing`
     Read more about methods for sharing **read** access to private sets of documentation without requiring a login for Read the Docs or your SSO provider.
```

This is a really good tip. This is like the real advantage of using SSO only for authentication, since it gives the flexibility of adding outside guest contributors and readers very easily.

Perhaps worth also mentioning secret share links?

Reply:
Yea, I thought about sharing here, so definitely a good addition. `seealso` might be too heavy inline, but we'll see how it renders.